<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginWidget</class>
 <widget class="QWidget" name="loginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/icon.svg</normaloff>:/icon.svg</iconset>
  </property>
  <widget class="QLabel" name="icon">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="icon.qrc">:/icon.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="key">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>选择LANChat.key文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>453</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>21</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SignupButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>注册用户</string>
   </property>
  </widget>
  <widget class="QLabel" name="askLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>没有 LANChat.key 文件？点击</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
